code/dynamic_programming/src/fibonacci/fibonacci.cpp

```diff
@@ -0,0 +1,42 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include <string>
+#include <sstream>
+#include <iostream>
+
+using namespace std;
+
+int dynamic_programming(int num);
+
+int main(){
+
+  int number;
+
+  cout << "Please enter an integer: ";
+  cin >> number;
+
+  int dyn_prog = dynamic_programming(number);
+
+  cout << "Fibonacci number is: " << dyn_prog << endl;
+
+  return 0;
+}
+
+int dynamic_programming(int num){
+
+  int fibarray[num+1];
```

Comment:
C++ does not support variable length arrays without compiler extensions. Use dynamic allocation (or my recommendation: std::vector)